src/views/studio/EditVideoSheet/EditVideoForm/EditVideoForm.tsx

```diff
@@ -519,7 +520,8 @@ export const EditVideoForm: React.FC<EditVideoFormProps> = ({
               )}
             />
           </FormField>
-          <FormField
+          <StyledLastFormField
```

Comment:
Not a big fan of that solution, but as you said in the chat - didn't find any better. Maybe you could add some comment explaining why `StyledLastFormField` exists? Also, moving that e.g. above will break that solution, right? Then we should include that in the comment